### PR TITLE
A first attempt to test visualization code

### PR DESCRIPTION
**Pull request by Tortar**
This is very rough, but at least we have something, so in my opinion it is nonetheless a good thing. Partially address #864 

Notice that I needed to change "Zombie" in "Zombiee" in `jld2tests.jl` because it conflicted with the "Zombie" in the examples folder. This is caused by the current design of the agent macro which always evaluates the quote at the module scope. We should fix this

### CODE REVIEW

**Comment by Tortar** [comment]
As expected we got a big boost in coverage

File: Project.toml
Comment (Datseris) [inline Project.toml:1]: Ah, yeah. We need to create a dedicated `test/Project.toml` file and move all of this stuff there (see for example how we do this here: https://github.com/JuliaDynamics/ComplexityMeasures.jl ). 

The `[extras]` is a deprecated way to add test-related dependencies.

**Review by Datseris — COMMENTED** [review]
Evaluating code coverage in the plots folder was a mistake. I didn't realize it. We need to remove this, irrespectiveily of whether we test the plotting or not.

See here for how to remove folders from the coverage: https://github.com/JuliaDynamics/Attractors.jl/blob/main/codecov.yml

File: Project.toml
Comment (Tortar) [reply Project.toml:1]: done!

**Comment by Tortar** [comment]
I think everything should be fine now (if tests passes)

**Comment by Tortar** [comment]
ok I probably need to test it locally instead :D

**Comment by Tortar** [comment]
Actually now the boost is even better :D

**Pull request merged**